include/swift/AST/DiagnosticsSema.def

```diff
@@ -812,7 +812,9 @@ ERROR(serialization_allowing_error_type,none,
       "allowing deserialization of error type '%0' in module '%1'",
       (StringRef, StringRef))
 WARNING(serialization_malformed_sourceinfo,none,
-      "unable to use malformed module source info '%0'", (StringRef))
+      "'%0' is either malformed or generated by a different Swift version. "
+      "Note that it is uses an unstable format and may leak internal project "
```

Comment:
typo, extra 'is'

Reply:
Argh, thanks. I originally had put up "is" and decided to change it to "uses". Not sure how I missed that I hadn't replaced it!